src/main/java/com/woodencloset/signalbot/Main.java

```diff
@@ -29,6 +29,10 @@ public static void main(String[] args) throws IOException, InvalidKeyException,
         commands.addOption(new Option("t", "test", false, "Test all responders using text input"));
         options.addOptionGroup(commands);
 
+        OptionGroup commands_ = new OptionGroup();
```

Comment:
Can you use options.addOption() directly without creating an OptionGroup first?

Reply:
https://github.com/aaronetz/signal-bot/pull/4/commits/1b6b694ede8046fbad7dfd43884a771743c0d44c